src/provable/transaction-leaves-bigint.ts

```diff
@@ -1,13 +1,16 @@
-import { Field, Bool, UInt32, UInt64, Sign } from "./field.js";
-import * as Json from "./gen/transaction-json.js";
-import { PublicKey } from "./curve.js";
 import {
-  ProvableExtended,
-  dataAsHash,
-  provable,
+  Field,
```

Comment:
this file is integrated into snarkyjs by this PR and should be reviewed in its entirety!